ssm_jax/linear_gaussian_ssm/models.py

```diff
@@ -455,25 +435,53 @@ def emap_step(_params):
         
         self.params = _params
         return jnp.array(log_probs)
+    
+    def fit_sgd(self, batch_emissions,
```

Comment:
This seems identical to https://github.com/probml/ssm-jax/blob/main/ssm_jax/hmm/models/base.py#L184.
Can we move `fit_sgd` up to the parent SSM base class or are there some differences?